.github/ISSUE_TEMPLATE/usability.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Usability Improvement
+about: Make Bevy more fun to use!
+title: ''
+labels: C-Usability, S-Needs-Triage
+assignees: ''
+---
+
+## What tripped you up?
```

Comment:
I think "tripped up" is a phase/idiom that some people won't fully grok (especially if they aren't fluent in english).

```md
## What usability issue did you encounter?
```

Reply:
Yeah, I was worried about that too. I don't love repeating the same phrase verbatim, because it doesn't help clarify "what is usabilty" to the reader.